install/Create_System_Configuration_Using_SHCD.md

```diff
@@ -0,0 +1,165 @@
+
+   - For new installations of CSM that have no prior seed files, each one must be created:
+
+      - [Create `application_node_config.yaml`](create_application_node_config_yaml.md)
+      - [Create `cabinets.yaml`](create_cabinets_yaml.md)
+      - [Create `hmn_connections.json`](create_hmn_connections_json.md)
+      - [Create `ncn_metadata.csv`](create_ncn_metadata_csv.md)
+      - [Create `switch_metadata.csv`](create_switch_metadata_csv.md)
+
+   - For re-installations of CSM 1.3, the previous seed files may be used and this step can be skipped.
+   - For new installations of CSM 1.3 that have prior seed files from CSM 1.2 or older, the previous seed files
+     may be used **except that the following files must be recreated** because of content or formatting changes:
+
+      - [Create `cabinets.yaml`](create_cabinets_yaml.md)
+      - [Create `hmn_connections.json`](create_hmn_connections_json.md)
+
+1. (`pit#`) Confirm that the following files exist.
+
+   ```bash
+   ls -l "${PITDATA}"/prep/{application_node_config.yaml,cabinets.yaml,hmn_connections.json,ncn_metadata.csv,switch_metadata.csv}
+   ```
+
+   Expected output may look like:
+
+   ```text
+   -rw-r--r-- 1 root root  146 Jun  6 00:12 /var/www/ephemeral/prep/application_node_config.yaml
+   -rw-r--r-- 1 root root  392 Jun  6 00:12 /var/www/ephemeral/prep/cabinets.yaml
+   -rwxr-xr-x 1 root root 3768 Jun  6 00:12 /var/www/ephemeral/prep/hmn_connections.json
+   -rw-r--r-- 1 root root 1216 Jun  6 00:12 /var/www/ephemeral/prep/ncn_metadata.csv
+   -rw-r--r-- 1 root root  150 Jun  6 00:12 /var/www/ephemeral/prep/switch_metadata.csv
+   ```
+
+### 3. Customize `system_config.yaml`
+
+1. (`pit#`) Create or copy `system_config.yaml`.
+
+   - If one does not exist from a prior installation, then create an empty one:
+
+      ```bash
+      csi config init empty
+      ```
+
+   - Otherwise, copy the existing `system_config.yaml` file into the working directory and proceed to the [Run CSI](#4-run-csi) step.
+
+1. (`pit#`) Edit the `system_config.yaml` file with the appropriate values.
+
+   > **NOTE**:
+   >
+   > - For a short description of each key in the file, run `csi config init --help`.
+   > - For more description of these settings and the default values, see
+   >   [Default IP Address Ranges](../introduction/csm_overview.md#2-default-ip-address-ranges) and the other topics in
+   >   [CSM Overview](../introduction/csm_overview.md).
+   > - To enable or disable audit logging, refer to [Audit Logs](../operations/security_and_authentication/Audit_Logs.md)
+   >   for more information.
+   > - If the system is using a `cabinets.yaml` file, be sure to update the `cabinets-yaml` field with `'cabinets.yaml'` as its value.
+
+   ```bash
+   vim system_config.yaml
+   ```
+
+### 4. Run CSI
+
+1. (`pit#`) Generate the initial configuration for CSI.
+
+   This will validate whether the inputs for CSI are correct.
+
+   ```bash
+   csi config init
+   ```
```

Comment:
If a section consists of a single step, we don't enumerate it.

````suggestion
(`pit#`) Generate the initial configuration for CSI. This will validate whether the inputs for CSI are correct.

```bash
csi config init
```
````